didcomm_messaging/routing.py

```diff
@@ -28,6 +28,8 @@ async def _resolve_services(self, to: str) -> List[DIDCommV2Service]:
         services = []
         if did_doc.service:  # service is not guaranteed to exist
             for did_service in did_doc.service:
+                if did_service.type != "DIDCommMessaging":
+                    continue
```

Comment:
Technically, `did-communication` could be used to identify a DIDComm v2 endpoint. We could consider supporting it if the `accept` property has `didcomm/v2` in it. I haven't fully formed an opinion either way on this yet. Thoughts?

Reply:
Since the DIDComm spec encourages DIDCommMessaging, I feel that we should encourage it as well. You brought up an excellent point of making the class more modular in the future in slack, which I think is an excellent idea. Though I'm not sure when we should do that